### PR TITLE
feat: integrate new sdk core

### PR DESCRIPTION
**Pull request by suevalov**
## Summary

Integrates https://github.com/contentful/contentful-sdk-core/pull/112

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,6 +124,9 @@
     "webpack-bundle-analyzer": "^3.3.2",
     "webpack-cli": "^3.1.2"
   },
+  "relativeDependencies": {
+    "contentful-sdk-core": "../contentful-sdk-core"
```
Comment (suevalov) [inline package.json:128]: Delete before merging

File: package.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     "tonic-example.js"
   ],
   "dependencies": {
-    "contentful-sdk-core": "^6.4.5",
+    "contentful-sdk-core": "^6.6.0",
     "axios": "^0.21.0",
```
Comment (marcolink) [inline package.json:59]: Possible to remove `axios` as direct dependency?

Comment (suevalov) [reply package.json:59]: no, it's still used in a couple places

**Review by marcolink — APPROVED** [review]
LGTM

**Pull request merged**